### PR TITLE
Added HTML class filters for feature/4813-form-html-classes

### PR DESCRIPTION
**Pull request by paddelboot**
Not sure if the filter name 'pods-field-html-class' follows any standards, feel free to change.

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Please reapproach this by setting a variable above it, filtering that variable, adding our default class to the end of it, and then `echo esc_attr( $classes );` into place.

Also please add a changelog in readme.txt, if there is not already a 2.7.2 line item, you can add a temporary one.

**Comment by paddelboot** [comment]
@sc0ttkclark Updated.

**Comment by pglewis** [comment]
Also, we'll need to merge up a fix for tests once that's done and merged into 2.x: #4817

File: ui/front/view.php [hunk 1]
```diff
@@ -9,32 +9,35 @@
...
+			if( isset( $field[ 'custom_display' ] ) && is_callable( $field[ 'custom_display' ] ) ) {
 				$value = call_user_func_array(
-					$field['custom_display'], array(
-						$pod->row(),
-						$pod,
-						$pod->field( $field['name'] ),
-						$field['name'],
-						$field,
-					)
+						$field[ 'custom_display' ], array(
```
Comment (pglewis) [inline ui/front/view.php]: Some rogue spacing in a few places still

**Comment by pglewis** [comment]
2.x has tests fixed so merging 2.x up to your branch will take care of the failing tests.  

Other than that, just a few small whitespace issues.

**Comment by paddelboot** [comment]
You want me to go over the code and merge the 2.x branch or is this in your hands now?

**Comment by pglewis** [comment]
It would be easiest if you have the time to patch it up since the PR is in a fork.

**Comment by paddelboot** [comment]
Done. Let me know when there's anything else.

**Comment by pglewis** [comment]
Excellent, looks good at a glance.  Assuming I don't find anything glaring with a closer look I should get this merged it today/tonight.

**Comment by paddelboot** [comment]
No problem. Also, to showcase a little bit what can be done with the new class filter: here's a screenshot of a form where I was able to apply the Bootstrap grid.

http://i65.tinypic.com/29ks31s.png

**Pull request merged**